src/sentry/static/sentry/app/components/hovercard.jsx

```diff
@@ -52,8 +52,16 @@ class Hovercard extends React.Component {
     this.hoverWait = setTimeout(() => this.setState({visible, rendered}), timeout);
   };
 
+  isOneLine() {
+    if (!this.cardElement || !this.state.visible) return null;
+    // spans will create a new bounding box for each line of text
+    // if the text wraps, so this is a way to test for text wrapping
+    return this.cardBodySpan.getClientRects().length <= 1;
```

Comment:
Neat. I've only used this on SVG elements before. I didn't know it worked on standard DOM nodes as well.

Reply:
I actually didn't know about this at all! I stole it from Katie's branch.